recipes/jupyterlab-accessible-themes/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "jupyterlab-accessible-themes" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1a8976f3c81b8be3b1b669de8aa4570718c1631df226f3a8850e6d67aee59089
+
+build:
+  noarch: python
+  number: 0
+
+requirements:
+  build:
+    - nodejs
+    - yarn
```

Can these two be on host instead of build? Usually build dependencies only need to be on "build" when cross-compiling but I may be wrong here.

```suggestion
    - nodejs
    - yarn
```

---

These dependencies are not going to be used at runtime, but they are needed for build. That's why I only added them in "build". 

Let me know if I should do the change regardless.

---

The "build" and "host" sections are both for building. Only run are for runtime dependencies.  Can you try to move those two to `host` and see how it goes here?

---

It worked!